docs/conceptual/fsharp-interactive-[fsi.exe]-reference.md

```diff
@@ -15,10 +15,10 @@ ms.assetid: 36af8d1b-dc08-4a37-9497-d23c0a0ac11c
 F# Interactive (fsi.exe) is used to run F# code interactively at the console, or to execute F# scripts. In other words, F# interactive executes a REPL (Read, Evaluate, Print Loop) for the F# language.
 
 
-## Interactive Programming with F# #
+## Interactive Programming with F#
```

Comment:
I believe the extra "#" removed here is necessary to get markdown to show "F#", isn't it?